mono/mini/mini.h

```diff
@@ -851,6 +849,20 @@ enum {
 #define inst_phi_args   data.op[1].phi_args
 #define inst_eh_blocks	 data.op[1].exception_clauses
 
+/* Return the lower 32 bits of the 64 bit immediate in INS */
+static inline guint32
+ins_get_l_low (MonoInst *ins)
+{
+	return (guint32)(ins->data.i8const & 0xffffffff);
+}
+
+/* Return the higher 32 bits of the 64 bit immediate in INS */
+static inline guint32
+ins_get_l_high (MonoInst *ins)
+{
+	return (guint32)((ins->data.i8const >> 32) & 0xffffffff);
+}
+
```

Comment:
inst_ls_word/inst_ms_word respected byte order, this doesn't

Reply:
They needed byte order because the i8const field is in a union with the data.op[0]/op[1] fields, this is not needed any more, since we directly read i8const.